main.py

```diff
@@ -1,11 +1,31 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
-
-
-if __name__ == "__main__":
-    main()
+    with open("players.json") as players_data:
+        players_info = json.load(players_data)
+    for player_name, player_data in players_info.items():
+        player = Player(
+            nickname=player_name,
+            email=player_data["email"],
+            bio=player_data["bio"],
+            race=Race.objects.get_or_create(
+                name=player_data["race"]["name"],
+                defaults={"description": player_data["race"]["description"]}
+            )[0],
+            guild=Guild.objects.get_or_create(
+                name=player_data["guild"]["name"],
+                description=player_data["guild"]["description"],
+                defaults={"description": player_data["guild"]["description"]}
```

Comment:
The `get_or_create` method for the Guild model should not include `description` in both the `get` and `defaults` parameters. You should only include it in `defaults` to ensure it is used only when creating a new Guild. Consider removing `description=player_data['guild']['description'],` from line 22.